docs/reference/stable-patch-format.rst

```diff
@@ -27,46 +27,51 @@ Every patch submitted to a stable kernel **must** have have its subject line
 starting with "[SRU]" followed by the release name against which the
 patch is targeted.
 
-The release name **must** be enclosed in "[ ]" brackets and can be
-abbreviated using the first letter of the release name (e.g. "B" for
-"Bionic"). For example:
+The release name **must** be enclosed in "[ ]" brackets and **should** be
+abbreviated using the first letter of the release name (e.g. "N" for
+"Noble"). For example:
 
 .. code-block:: none
 
-   [SRU][Focal][PULL] Focal update: v5.4.56 upstream stable release
+   [SRU][N][PATCH 0/1] Fix error of resume on rtl8168fp
 
 * If a patch is to be applied to multiple releases, a list of release names
-  must be provided. For example:
+  must be provided. For example, when it targets Bionic and Focal:
```

Comment:
```suggestion
  must be provided, with "/" separating the letter representing each series. For example, when it targets Bionic and Focal:
```

Reply:
I know this is mentioned below but that is in a notebox which should supplement the main content and "theoretically" can be skipped.
I suggest just making it explicitly clear here.